types/relay-runtime/lib/store/RelayRecordSource.d.ts

```diff
@@ -2,20 +2,19 @@ import { MutableRecordSource, RecordMap, Record } from './RelayStoreTypes';
 import { DataID } from '../util/RelayRuntimeTypes';
 import { RecordState } from './RelayRecordState';
 
-export class RelayRecordSource {
+export class RelayRecordSource implements MutableRecordSource {
```

Comment:
Why this class implements `MutableRecordSource`?

https://github.com/facebook/relay/blob/v6.0.0/packages/relay-runtime/store/RelayRecordSource.js#L19

Reply:
flow does https://github.com/facebook/relay/blob/v6.0.0/packages/relay-runtime/store/RelayRecordSource.js#L20

but TS doesn't support ctor return types: https://github.com/Microsoft/TypeScript/issues/27594

we _could_ just use pure structural typing but implementing the interface means we get earlier errors if we forget to specify a method